docs/pseudo_syscalls.md

```diff
@@ -1,15 +1,24 @@
-# Syzkaller pseudo-syscalls
+# Pseudo-syscalls
 
 Besides regular system calls, a [syscall
 description](syscall_descriptions.md) file can also contain
-pseudo-syscalls. These are C functions defined in the syzkaller
```

Comment:
I've also removed some uses of "syzkaller" b/c everything in syzkaller is syzkaller-something. So in most cases "syzkaller" is redundant and would make sense, say, if we publish it on an external site.

Reply:
Done